test/test_utils_tests.cc

```diff
@@ -208,3 +208,16 @@ TEST_CASE("most operations fail when called on an empty query", "[command_query]
 	CHECK_THROWS_WITH(query.have_successors(command_query_testspy::create_for({})), "Operation 'have_successors' not allowed on empty query");
 	CHECK_THROWS_WITH(query.have_type(command_type::epoch), "Operation 'have_type' not allowed on empty query");
 }
+
+TEST_CASE("tests that log any message in excess of level::info fails by default", "[test_utils][log][!shouldfail]") { CELERITY_WARN("spooky message!"); }
```

Comment:
Should be either "test that logs" or "fail".

Reply:
You're right! Fixed.